src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,36 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int DATE = 0;
+    private static final int NAME = 1;
+    private static final int WORKING_HOURS = 2;
+    private static final int INCOME_PER_HOUR = 3;
+    private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate from = LocalDate.parse(dateFrom, formatter);
+        LocalDate to = LocalDate.parse(dateTo, formatter);
+        StringBuilder info = new StringBuilder("Report for period " + dateFrom + " - " + dateTo);
+
+        for (String employee : names) {
+            int workerSalary = 0;
+
+            for (String elm : data) {
```

Comment:
```suggestion
            for (String info : data) {
```

Just use something like this. Better not to use elm or similar shortcuts